charts/streams-app/templates/deployment.yaml

```diff
@@ -115,9 +115,9 @@ spec:
             - name: "{{ .Values.configurationEnvPrefix }}_VOLATILE_GROUP_INSTANCE_ID"
               value: "true"
           {{- end }}
-          {{- if hasKey .Values.streams "brokers" }}
-            - name: "{{ .Values.configurationEnvPrefix }}_BROKERS"
-              value: {{ .Values.streams.brokers | quote }}
+          {{- if hasKey .Values.streams "bootstrapServers" }}
+            - name: "{{ .Values.configurationEnvPrefix }}_BOOTSTRAP_SERVER"
```

Comment:
The cli param for all kafka cli tools is always `--bootstrap-server`. However it is `StreamsConfig.BOOTSTRAP_SERVERS_CONFIG`. Thats why I named it bootstrapServers in the chart. wdyt?

Reply:
I would prefer if we're consistent and then stick to using the property names everywhere, so `bootstrap.servers` in this case.

Reply:
So you would use `--bootstrap-servers`?

Reply:
yes

Reply:
I added an alias to have both available